rcl_logging_interface/test/test_get_logging_directory.cpp

```diff
@@ -14,17 +14,19 @@
 
 #include <iostream>
 #include <string>
+#include <filesystem>
 
 #include "gtest/gtest.h"
 
-#include "rcpputils/filesystem_helper.hpp"
 #include "rcpputils/env.hpp"
 #include "rcutils/allocator.h"
 #include "rcutils/env.h"
 #include "rcutils/error_handling.h"
 
 #include "rcl_logging_interface/rcl_logging_interface.h"
 
+namespace fs = std::filesystem;
```

Comment:
I'd really prefer not to do this everywhere; we use `std::` in most of the codebase, so let's just spell out `std::filesystem` everywhere.